en/08_Loading_models.adoc

```diff
@@ -5,47 +5,20 @@
 == Introduction
 
 Your program is now ready to render textured 3D meshes, but the current geometry in the `vertices` and `indices` arrays is not very interesting yet.
-In this chapter we're going to extend the program to load the vertices and indices from an actual model file to make the graphics card actually do some work.
+In this chapter, we're going to extend the program to load the vertices and indices from an actual model file to make the graphics card actually do some work.
 
 Many graphics API tutorials have the reader write their own OBJ loader in a chapter like this.
 The problem with this is that any remotely interesting 3D application will soon require features that are not supported by this file format, like skeletal animation.
 We _will_ load mesh data from an OBJ model in this chapter, but we'll focus more on integrating the mesh data with the program itself rather than the details of loading it from a file.
 
-== Library
```

Comment:
Any reason this was completely removed? I think we should at least keep the mention of tinyobj and where it's coming from.

Reply:
I moved it to the development environment setup chapter.  I'll add a section back here to refer back to that chapter incase they get this far and don't yet have it.